drizzle/0027_complete_iron_fist.sql

```
@@ -0,0 +1,5 @@
+CREATE INDEX "space_repetition_userId_index" ON "grumma"."space_repetition" USING btree ("userId");
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)